app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
```

Comment:
[REQUIRED] You don't need  `foreign_key`. It is usually added in has_many. Also, you have `belongs_to :user, class_name: 'User', foreign_key: 'user_id'` but your `has_many :comments` still has a `foreign_key` pointing to author. It is either you stick to author_id or remove the foreign_key in the user model and stick to user_id. I think you should read up on associations